web/package.json

```
@@ -12,6 +12,7 @@
   "devDependencies": {
     "@types/d3": "7.4.0",
     "@types/luxon": "3.0.0",
+    "@types/papaparse": "^5.3.3",
```

```suggestion
    "@types/papaparse": "5.3.3",
```


Since enabling Renovate, we are pinning `devDependencies`.